e2e/pages/events/uploadC2DocumentsEvent.page.js

```diff
@@ -12,6 +12,17 @@ module.exports = {
     pbaNumber: '#temporaryC2Document_pbaNumber',
     clientCode: '#temporaryC2Document_clientCode',
     customerReference: '#temporaryC2Document_fileReference',
+    documentName: '#temporaryC2Document_manageDocumentBundle_0_name',
+    documentNotes: '#temporaryC2Document_manageDocumentBundle_0_notes',
+    documentDateAndTime: {
+      day: 'temporaryC2Document_manageDocumentBundle_0_dateTimeReceived-day',
+      month: 'temporaryC2Document_manageDocumentBundle_0_dateTimeReceived-month',
+      year: 'temporaryC2Document_manageDocumentBundle_0_dateTimeReceived-year',
+      hour: 'temporaryC2Document_manageDocumentBundle_0_dateTimeReceived-hour',
+      minute: 'temporaryC2Document_manageDocumentBundle_0_dateTimeReceived-minute',
+      second: 'temporaryC2Document_manageDocumentBundle_0_dateTimeReceived-second',
+    },
+    uploadC2SupportingDocument: '#temporaryC2Document_manageDocumentBundle_0_document',
```

Comment:
This might be nicer having 'supportingDocument' or similar as a parent key. Then we could have

'SupportingDocument': {
  'name': '',
  'notes': ''
  etc etc
}

Reply:
done